rules-reviewed/openjdk11/openjdk8/java-removals.windup.xml

```diff
@@ -79,5 +79,23 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="java-removals-00101">
+            <when>
+                <or>
+                    <javaclass references="sun.misc.BASE64Encoder"/>
+                    <javaclass references="sun.misc.BASE64Decoder"/>
+                </or>
```

Comment:
I think it would be great to try to have just one condition with a `where` parameter to "inject" `Encoder` or `Decoder` and use the same also to drive the hint's message below.